Controller.cpp

```diff
@@ -443,22 +357,170 @@ namespace {
 }; //namespace
 
 namespace Procon {
+	using std::array;
+
+	void SetDefaultCalibration(CalibrationData &dat) {
+		constexpr uchar ucmin = std::numeric_limits<uchar>::min();
+		constexpr uchar ucmax = std::numeric_limits<uchar>::max();
+		dat.leftCenter.x = ucmax / 2;
+		dat.leftCenter.y = ucmax / 2;
+		dat.rightCenter = dat.leftCenter;
+		dat.left.x.min = dat.leftCenter.x;
+		dat.left.x.max = dat.leftCenter.x;
+		dat.left.y = dat.left.x;
+		dat.right = dat.left;
+	}
```

Comment:
All of the calibration values are `uint16_t` now, right? Shouldn't the defaults be updated?

Reply:
I think they should be updated, but I'm not sure whether the ADCs for the analog sticks are 12 or 14 bit. Perhaps @CTCaer can tell us something about this?

Reply:
The analog stick values and calibration values are all unsigned 12bit.

The motion sensor values are all signed 16bit.